docker/.env

```diff
@@ -331,12 +331,11 @@ MG_UI_IMAGE_URL=http://localhost:9097
 MG_UI_BASEURL=http://localhost:3000
 
 #Customer support email variables
-SUPPORT_EMAIL=
-SUPPORT_EMAIL_PASS=
+MG_SUPPORT_EMAIL=
+MG_SUPPORT_EMAIL_PASS=
 
 # Docker image tag
 MG_RELEASE_TAG=latest
 
-# runtime environment
-RUNTIME_ENV=production # to be changed to production before production deployment
-MG_UI_DOCKER_ACCEPT_EULA=false
+# Set to yes to accept the EULA agreement for the UI services. To view the UELA agreement visit: https://github.com/absmach/eula
```

Comment:
EULA, not UELA - fix typo.

`To view the UELA agreement` - remove `agreement`, it is already part of EULA